src/api/spec/services/meta_controller_service/project_updater_spec.rb

```diff
@@ -0,0 +1,91 @@
+require 'rails_helper'
+
+RSpec.describe ::MetaControllerService::ProjectUpdater do
+  let(:project) { create(:project, name: 'openSUSE_41') }
+  let(:admin_user) { create(:admin_user, login: 'Admin') }
+
+  before do
+    User.current = admin_user
+  end
+
+  context 'with valid meta' do
+    let(:meta) do
+      <<-HEREDOC
+      <project name="#{project.name}">
+      <title/><description/>
+      </project>
+      HEREDOC
+    end
+
+    let(:project_updater) { ::MetaControllerService::ProjectUpdater.new(project: project, request_data: Xmlhash.parse(meta)) }
+
+    describe '#new' do
+      it { expect(project_updater).not_to be_nil }
+    end
+
+    describe '#call' do
+      it { expect { project_updater.call }.not_to raise_error }
```

Comment:
This makes me wonder when this method would raise an exception and which one. I don't see any test for this.

Reply:
looking the code you will that is a transaction. https://github.com/vpereira/open-build-service/blob/master/src/api/app/services/meta_controller_service/project_updater.rb#L16

Reply:
IMO you can drop this test. The test for `valid?` already ensures that `call` does not raise an exception and since there isn't any particular exception we would expect here I don't see that point of having a test for this.

Reply:
did it @bgeuken